generators/app/templates/src/app/components/RadioButton/index.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import styles from './styles';
 
 const RadioButton = ({ onPress, ...props }) => (
-  <button type='button' style={[styles.base, props.style]} onClick={onPress}>
+  <button type="button" style={[styles.base, props.style]} onClick={onPress}>
```

Comment:
Shouldn't this receive `...props` ?

Reply:
I'd make this API parallel, by renaming `onPress` to `onClick`

Reply:
This'll be erased in a future PR